app/src/app/components/routes/ProtectedRoute.tsx

```diff
@@ -18,15 +19,29 @@ export default function ProtectedRoute() {
     useEffect(() => {
         const user = getCurrentUser();
 
-        if (authConfig.enableAuth) {
-            if (!isAuthenticated && !user?.token) {
+        if (!authConfig.enableAuth) {
+            return;
+        }
+
+        if (!isAuthenticated && !user?.token) {
```

Comment:
What case does this cover, where we've authenticated but there isn't a user token? Could you add a comment to describe please?